docs/project/arch-diagram.md

```
@@ -0,0 +1,128 @@
+  External Services (CI/CD)
+  ─────────────────────────
+  GitHub ──▶ GitHub Actions ──▶ Docker Hub
+                    │
+                    ▼
+              CodeDeploy Agent (on EC2)
+```
+
+## Resource Summary
+
+| Resource | Name | Spec |
+|----------|------|------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)